macros/customizeLaTeX.pl

```diff
@@ -0,0 +1,103 @@
+=head1 NAME
+
+customizeLaTeX.pl - Defines default LaTeX constructs for certain mathematical
+                    ideas.
+
+=head1 DESCRIPTION
+
+The functions are loaded by default.  Any/all can be overridden 
+in your course's PGcourse.pl
+=cut
+
+sub _customizeLaTeX_init {
+
+} #prevents this file from being loaded twice.
+
+##### Set theory macros
+sub set_minus{ 
+	#return "\\setminus";
+	return '-';
+};
+
+#####  Logic macros
+sub negate { 
+	return "\\mathbin{\\sim}";
+	#return "\\lnot";
+};
+
+sub implies {
+	return "\\implies";
+	#return "\\Rightarrow";
+}
+
+##### Linear algebra macros
+
+sub vectorstyle {
+	my $v = shift;
+	#return "\\vec\{$v\}"
```

Comment:
I rewrote this as 

sub vectorstyle {
	my $v = shift;
	#return "\\vec\{$v\}";
	 return "\\vec{$v} ";
	#return "$v";
}

so that the default value shows some change in the style of the vector.  As far as I can tell the backslash before the braces is unnecessary although it does no harm in PGML.

Reply:
there are double backslashes before vec. (the HTML wipes them out).  There were single backslashes before the braces in the 3rd line.